src/components/HeaderBar.js

```diff
@@ -46,39 +49,51 @@ class HeaderBar extends React.Component {
         window.removeEventListener( "resize", this.updateBarOpen );
         document.removeEventListener( 'click', this.handleClickOutside );
     }
+    
+
+    handleDarkModeChanged(){
+        this.props.callbackParent(this.state.darkMode);
+    }
+
 
     render() {
+
         const items = (
+
             <div>
                 <NavLink activeStyle={{textShadow: "2px 2px 5px #5DADE2"}} exact to="/">
                     <li>
-                        <i className="fas fa-home">&nbsp;Home</i>
+                        <i className={this.props.darkMode ? "fas fa-home dark-mode-icon " : "fas fa-home " }>&nbsp;Home</i>
```

Comment:
Could also use the ${} format for this too

Reply:
Ok, I'll change all of them later. Since some places just use one class, so don't really need to change to that format haha

Reply:
All changed